go.mod

```diff
@@ -1,13 +1,11 @@
 module github.com/aquasecurity/trivy-checks
 
-go 1.24
-
-toolchain go1.24.0
+go 1.24.2
 
 require (
-	github.com/aquasecurity/trivy v0.61.1-0.20250407075540-f1329c7ea1aa
+	github.com/aquasecurity/trivy v0.62.1-0.20250528213022-4a7ebb70b48f
```

Comment:
@nikpivkin I've updated to the latest trivy but yet in the CI it's complaining about lack of the `raw-config-scanners` flag. Any ideas why?

It briefly worked for me locally prior to rebasing the branch. Not sure why would that cause this to happen.

Reply:
I see - yeah I think we can merge and cut a release then.